pkg/controller/certificates/sync.go

```diff
@@ -59,6 +60,11 @@ const (
 	messageCertificateIssued  = "Certificate issued successfully"
 	messageCertificateRenewed = "Certificate renewed successfully"
 	messageRenewalScheduled   = "Certificate scheduled for renewal in %d hours"
+
+	altNamesAnnotation   = "certmanager.k8s.io/alt-names"
+	commonNameAnnotation = "certmanager.k8s.io/common-name"
+	issuerNameAnnotation = "certmanager.k8s.io/issuer-name"
+	issuerKindAnnotation = "certmanager.k8s.io/issuer-kind"
```

Comment:
Can you move these into `pkg/apis/certmanager/v1alpha1/types.go` at the top of the file as consts, export them, and suffix each with `Key`?